_posts/2016-06-23-census-centrality.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Crowdsensing the Census - Exploring Street Network"
+date:   2016-07-20
+author: Carlos Espino
+---
+
+Last week we wanted to extract more information from Milan´s OSM data, so we performed visualization and analysis of its street network. We aim to find a relationship between the street connectivity and social deprivation. We also wanted to further investigate how this relationship differs from that of Milan in a city belonging to a developing country, particularly Mexico City.
+
+In order to compute connectedness of the street network, we tried the two following graph centrality measures: 
+
++ _Betweenness centrality_ measures the number of shortest paths between any given pair of nodes that pass through an individual node.  
+
++ _Closeness centrality_ measures how "close" you are to the center of the graph- that is, the length of average shortest path between a node and all nodes to the graph. For each node, the algorithm computes the length of the average shortest path between it and all the other nodes in the graph and calculates the inverse total length.
+
+In order to compute the centrality, we first converted the spatial lines' shapefile into a graph object, where the nodes in the graph correspond to the intersections, and the edges correspond to the streets. 
```

Comment:
OK - I now see you have it down here. I would change the order, to introduce this idea before the metrics are introduced.